test/elixir/lib/couch.ex

```diff
@@ -175,3 +175,109 @@ defmodule Couch do
     end
   end
 end
+
+defmodule Rawresp do
```

Comment:
couldn't we just write a new file for this? Too much pain? (at the moment I can't remember if we could call those functions inside the module this way)

Reply:
Sure, i'll put it in a new file

Really, a tried to do it with the current Couch module at couch.ex but I don't know how to parametrize this behaviour.

Reply:
Done